openapi/platform/platform.yaml

```diff
@@ -6807,19 +7735,12 @@ components:
                 - sentToPeer
               description: |-
                 draft - newly created proposal that hasn't been sent to anyone
-
                 sentToPeer - sent for internal review
-
                 approvedByPeer - internal review approved
-
                 sentToRecipient - proposal sent via email to contact of the account
-
                 viewedByRecipient - the contact of the account viewed the proposal
-
                 feedbackFromRecipient - the contact of the account left a comment on the proposal
-
                 approvedByRecipient - the contact of the account approved the proposal
-
```

Comment:
The blank lines in the description will need to be added back in. Since the description is treated as markdown if they are missing when displayed to users on the documentation site it will all appear as one line. 

There are a few other cases of this as well.